### DISCUSSION TITLE
What if you are not a Student but graduated and want to join the dev pack ?

### DISCUSSION THREAD

**Discussion by Vidhi-44** [community/community#58324] [GitHub Education]
### Hi everyone,

According to GitHub Octernship  eligiblity  criteria I do fit in the 2 things.
I have an active GitHub account , I am 13+ in age but I am just not enrolled in an institution so I cant apply for the student dev pack . 
But I really want to participate in the Octernship  so do let me know what should I do for this ?

**Comment by the-officialjosh** [comment]
If you're not a student, you can still request a regular GitHub for education discount.  But as you know clearly the program is reserved for those on the student pack as well as in the 10 locations. If possible try to get the pack by paying in with the discount if it is important you join

**Comment by samus-aran** [comment]
Hi @Vidhi-44 👋 

We're happy you are keen to join Octernships. However you must be 18 years or older to apply.

To apply for the Octernships program, you must follow the requirements listed below:

- 18+ years old
- A verified student on GitHub Global Campus
- An active contributor on GitHub
- Enrolled in an institution with work eligibility in one of the ten countries: India, Singapore, Indonesia, Malaysia, Vietnam, Philippines, Thailand, Mexico, Nigeria, and Colombia.

We do accept middle school enrolment to GitHub Global Campus though if you wish to apply for the Student Developer Pack. 👍
